docs/AboutUs.md

```diff
@@ -19,15 +19,16 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 * Role: Project Advisor
 
-### Jane Doe
+### Tan Juay Hee
 
 <img src="images/johndoe.png" width="200px">
```

Comment:
Should also upload own photo (or a placeholder) and rename the photo to your username (in lowercase), as the tracker also checks for this.

Reply:
agree